programs/address-lookup-table/src/processor.rs

```diff
@@ -462,7 +462,7 @@ impl Processor {
 
```

Comment:
While we are at it: How about `table_data.reserve()` before the `for` loop extending it multiple times?